flowise-embed/src/components/bubbles/GuestBubble.tsx

```diff
@@ -13,23 +14,52 @@ type Props = {
   backgroundColor?: string;
   textColor?: string;
   fontSize?: number;
+  renderHTML?: boolean;
 };
 
 const defaultBackgroundColor = '#3B81FC';
 const defaultTextColor = '#ffffff';
 const defaultFontSize = 16;
 
-Marked.setOptions({ isNoP: true });
-
 export const GuestBubble = (props: Props) => {
   let userMessageEl: HTMLDivElement | undefined;
 
+  Marked.setOptions({ isNoP: true, sanitize: props.renderHTML !== undefined ? !props.renderHTML : true });
+
   onMount(() => {
     if (userMessageEl) {
       userMessageEl.innerHTML = Marked.parse(props.message.message);
     }
   });
 
+  const renderFileUploads = (item: Partial<FileUpload>) => {
+    if (item?.mime?.startsWith('image/')) {
+      const fileData = `${props.apiHost}/api/v1/get-upload-file?chatflowId=${props.chatflowid}&chatId=${props.chatId}&fileName=${item.name}`;
+      const src = (item.data as string) ?? fileData;
+      return (
+        <div class="flex items-center justify-center max-w-[128px] mr-[10px] p-0 m-0">
+          <img class="w-full h-full bg-cover" src={src} />
+        </div>
+      );
+    } else if (item?.mime?.startsWith('audio/')) {
+      const fileData = `${props.apiHost}/api/v1/get-upload-file?chatflowId=${props.chatflowid}&chatId=${props.chatId}&fileName=${item.name}`;
+      const src = (item.data as string) ?? fileData;
+      return (
+        <audio class="w-[200px] h-10 block bg-cover bg-center rounded-none text-transparent" controls>
+          Your browser does not support the &lt;audio&gt; tag.
+          <source src={src} type={item.mime} />
+        </audio>
```

Comment:
Excelente! Só uma dúvida: a mensagem 'Your browser does not support the &lt;audio&gt; tag' poderia estar em português? Por exemplo, 'Seu navegador não suporta o player de áudio', para facilitar a compreensão do usuário. Ou, teria algum motivo específico para esta mensagem de fallback ser exibida dessa forma?